WORKSPACE

```diff
@@ -391,6 +394,11 @@ hazel_repositories(
         , ("shake", {"version": "0.17.8", "sha256": "ade4162f7540f044f0446981120800076712d1f98d30c5b5344c0f7828ec49a2"})
         , ("filepattern", {"version": "0.1.1", "sha256": "f7fc5bdcfef0d43a793a3c64e7c0fd3b1d35eea97a37f0e69d6612ab255c9b4b"})
         , ("terminal-progress-bar", {"version": "0.4.0.1", "sha256": "c5a9720fcbcd9d83f9551e431ee3975c61d7da6432aa687aef0c0e04e59ae277"})
+        # Deps for haskell LSP
+        , ("rope-utf16-splay" , {"version": "0.2.0.0", "sha256": "83d1961bf55355da49a6b55d6f58d02483eff1f8e6df53f4dccdab1ac49e101d"})
```

Comment:
Can we ask for this to be added to stackage.

Reply:
Yup, I can do that before release

Reply:
Can we do that now? It is likely to have some lag from the maintainer.